services/motion/builtin/builtin_test.go

```diff
@@ -1219,38 +1261,28 @@ func TestMoveOnGlobe(t *testing.T) {
 			MovementSensorName: injectedMovementSensor.Name(),
 			Heading:            -10000000,
 			Destination:        dst,
-			Extra:              extra,
+			MotionCfg: &motion.MotionConfiguration{
+				PlanDeviationMM: 1,
+			},
+			Extra: extra,
 		}
 		executionID, err := ms.MoveOnGlobe(ctx, req)
 		test.That(t, err, test.ShouldBeNil)
 		test.That(t, executionID, test.ShouldNotResemble, uuid.Nil)
 	})
-
 	t.Run("is able to reach a nearby geo point when the motion configuration is empty", func(t *testing.T) {
 		injectedMovementSensor, _, fakeBase, ms := createMoveOnGlobeEnvironment(ctx, t, gpsPoint, nil, 5)
 		defer ms.Close(ctx)
 		req := motion.MoveOnGlobeReq{
 			ComponentName:      fakeBase.Name(),
 			MovementSensorName: injectedMovementSensor.Name(),
 			Heading:            90,
-			Destination:        dst,
+			Destination:        geo.NewPoint(gpsPoint.Lat(), gpsPoint.Lng()+1e-4),
 			MotionCfg:          &motion.MotionConfiguration{},
-			Extra:              extra,
-		}
-		executionID, err := ms.MoveOnGlobe(ctx, req)
-		test.That(t, err, test.ShouldBeNil)
-		test.That(t, executionID, test.ShouldNotResemble, uuid.Nil)
-	})
-
-	t.Run("is able to reach a nearby geo point when the motion configuration nil", func(t *testing.T) {
```

Comment:
We still need to be able to reach a geo point when the motion configuration is nil

Reply:
can you please add this test case back?

Reply:
never mind. I was looking at an old commit